docs/build/start-building/interacting-with-the-network.md

```diff
@@ -28,20 +28,20 @@ Here are some scripts to set up your own remote networks for development:
 
 ## Devnet
 
-In order to make testing easier, a live devnet (developer network) is now available for use. It features 512MB sectors and reduced proofs parameters, so sealing is much faster than in the full Testnet version.
+In order to make testing easier, a live devnet (developer network) is now available for use. We call this network the Nerpa Devnet (the nerpa is a species of seal, and one of the smallest true seals). The Nerpa Devnet features 512MB sectors (in addition to 32GB and 64GB sectors) and reduced proofs parameters, so sealing is much faster than in the full Testnet version. This devnet is currently primarily intended for participants in the [HackFS hackathon](https://hackfs.com).
```

Comment:
This is intended to be a general-use devnet. All devgrant winners etc are also being pointed here. Can we delete the last sentence in 31?

Reply:
See #135.